OpenRA.Mods.Common/Widgets/UnitCommandWidget.cs

```diff
@@ -158,7 +168,8 @@ bool PerformStanceCycle()
 				return new Order("SetUnitStance", a, false) { ExtraData = (uint)nextStance };
 			});
 
-			Game.AddChatLine(Color.White, "Battlefield Control", "Unit stance set to: {0}".F(nextStance));
+			if (cycleStances)
```

Comment:
Should this only be for cycling stances. If a player hits the wrong key, they would not get any feedback until the actor(s) behave differently from the expected stance.